main.c

```diff
@@ -38,6 +38,16 @@ int main(int argc, char* argv[])
 	/*
 	 * TODO: Print if its a win or draw
 	*/
+	if(f==1)
```

Comment:
Can you change the name of the variable `f` to something else which describes the purpose it serves?